starter_templates/redis/rust/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "rustredis"
+version = "0.1.0"
+authors = ["Syed Ghufran Hassan <syedghufranhassan@gmail.com>"]
```

Comment:
Please change this to `CodeCrafters <hello@codecrafters.io>`

Reply:
Acknowledged sir